oracle.go

```diff
@@ -98,6 +98,7 @@ func (o *Oracle) CreateUser(ctx context.Context, statements dbplugin.Statements,
 	if err != nil {
 		return "", "", err
 	}
+	username = strings.ToUpper(username)
```

Comment:
This is fine, though we may want to log a Jira to update SQLCredentialsProducer with some fields to control this sort of thing. No need to change all that now though.

Reply:
We did the opposite of this in the Redshift engine, adding a `LowercaseUsername` attribute to creds producer. Agree with @kalafut here though, separate PR material.

Reply:
Agreed. I've been thinking the same thing.